cmd/prometheus-config-reloader/main.go

```diff
@@ -17,21 +17,32 @@ package main
 import (
 	"context"
 	"fmt"
+	"net/http"
 	"os"
 	"regexp"
 	"strings"
+	"time"
 
 	"github.com/prometheus-operator/prometheus-operator/pkg/version"
 
 	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
 	"github.com/oklog/run"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
 	"github.com/thanos-io/thanos/pkg/reloader"
 	"gopkg.in/alecthomas/kingpin.v2"
 )
 
 const (
-	logFormatLogfmt                     = "logfmt"
-	logFormatJson                       = "json"
+	logFormatLogfmt = "logfmt"
+	logFormatJson   = "json"
+
+	logLevelDebug = "debug"
+	logLevelInfo  = "info"
+	logLevelWarn  = "warn"
+	logLevelError = "error"
```

Comment:
We have none as well in prom-operator, is there a reason why we don't use it here as well? For consistency reasons is why I am mainly asking.

Reply:
I wasn't aware of the `none` level (Prometheus doesn't have it) but I agree that it's better to be consistent with what we have already.